moveit_plugins/moveit_simple_controller_manager/CMakeLists.txt

```diff
@@ -26,8 +29,6 @@ catkin_package(
   CATKIN_DEPENDS moveit_core actionlib control_msgs
     )
 
-include_directories(include)
```

Comment:
Just assuming the key of this PR is to move this line to somewhere before `catkin_package`?